tutorials/evoked/plot_20_visualize_evoked.py

```diff
@@ -0,0 +1,249 @@
+
+###############################################################################
+# For larger numbers of sensors, the method :meth:`evoked.plot_topo()
+# <mne.Evoked.plot_topo>` and the function :func:`mne.viz.plot_evoked_topo`
+# can both be used. The :meth:`~mne.Evoked.plot_topo` method will plot only a
+# single condition, while the :func:`~mne.viz.plot_evoked_topo` function can
+# plot one or more conditions on the same axes, if passed a list of
+# :class:`~mne.Evoked` objects. The legend entries will be automatically drawn
+# from the :class:`~mne.Evoked` objects' ``comment`` attribute:
+
+mne.viz.plot_evoked_topo(evokeds_list)
+
+###############################################################################
+# By default, :func:`~mne.viz.plot_evoked_topo` will plot all MEG sensors (if
+# present), so to get EEG sensors you would need to modify the evoked objects
+# first (e.g., using :func:`mne.pick_types`).
+#
+# .. note::
+#
+#     In interactive sessions, both approaches to topographical plotting allow
+#     you to click one of the sensor subplots to pop open a larger version of
+#     the evoked plot at that sensor.
+#
+#
+# 3D Field Maps
+# ^^^^^^^^^^^^^
+#
+# The scalp topographies above were all projected into 2-dimensional overhead
+# views of the field, but it is also possible to plot field maps in 3D. To do
+# this requires a :term:`trans` file to transform locations between the
+# coordinate systems of the MEG device and the head surface (based on the MRI).
+# You *can* compute 3D field maps without a ``trans`` file, but it will only
+# work for calculating the field *on the MEG helmet from the MEG sensors*.
+
+subjects_dir = os.path.join(sample_data_folder, 'subjects')
+sample_data_trans_file = os.path.join(sample_data_folder, 'MEG', 'sample',
+                                      'sample_audvis_raw-trans.fif')
+
+###############################################################################
+# By default, MEG sensors will be used to estimate the field on the helmet
+# surface, while EEG sensors will be used to estimate the field on the scalp.
+# Once the maps are computed, you can plot them with :meth:`evoked.plot_field()
+# <mne.Evoked.plot_field>`:
+
+maps = mne.make_field_map(evks['aud/left'], trans=sample_data_trans_file,
+                          subject='sample', subjects_dir=subjects_dir)
+evks['aud/left'].plot_field(maps, time=0.1)
+
+###############################################################################
+# You can also use MEG sensors to estimate the *scalp* field by passing
+# ``meg_surf='head'``. By selecting each sensor type in turn, you can compare
+# the scalp field estimates from each.
+
+mags = evks['aud/right'].copy().pick_types(meg='mag')
+grads = evks['aud/right'].copy().pick_types(meg='grad')
+eegs = evks['aud/right'].copy().pick_types(meg=False, eeg=True)
+sensor_dict = dict(magnetometers=mags, gradiometers=grads, EEG=eegs)
+
+for title, evk in sensor_dict.items():
```

Comment:
I woudl avoid the definition of mags, grads etc. variables and do:

```
for ch_type in ['mag', 'grad', 'eeg']:
    evk = evks['aud/right'].copy().pick(ch_type)
   ... 
```

then you can use ch_type as title.